core/controllers/acl_decorators.py

```diff
@@ -1963,14 +1916,10 @@ def can_user_publish_skill(user, skill_rights):
         Returns:
             bool. Whether the given user can publish the given skill.
         """
-
-        if skill_rights is None:
```

Comment:
Ditto

Reply:
This function(can_user_publish_skill) is called in test_can_publish_skill which already checks for skill rights to be None [here](https://github.com/oppia/oppia/blob/develop/core/controllers/acl_decorators.py#L1996)

Reply:
Same suggestion, please inline if possible.

Reply:
Done